.github/ISSUE_TEMPLATE/new-item.yml

```diff
@@ -0,0 +1,59 @@
+name: Suggest new item
+description: Suggest a new maturity item.
+labels:
+  - new maturity item
+body:
+  - type: checkboxes
+    attributes:
+      label: Precheck
+      description: Check that current maturity items can't be modified/generalized to include your concern.
+      options:
+        - label: This item can not be defined below any of the current maturity items.
```

Comment:
```suggestion
        - label: Yes.
```

Reply:
It's gonna look like this:
![Screenshot from 2022-02-20 12-58-25](https://user-images.githubusercontent.com/11619295/154836464-fb2bdfb0-1b79-4849-8dc7-cb002cffc286.png)

So I think the usage of `description` field is just to give a hint and the check box `lable` should be descriptive enough.